apps/studio/src/server/modules/collection/collection.router.ts

```diff
@@ -141,6 +202,16 @@ export const collectionRouter = router({
             }
             throw err
           })
+
+        await logResourceEvent(tx, {
+          eventType: "ResourceCreate",
+          by: user,
+          delta: {
+            before: null,
+            after: { resource: addedResource, blob },
+          },
+        })
+
         return addedResource
```

Comment:
question: the return object is a bit different now, should we `pick` just the `id`?

Reply:
sounds good

Reply:
actually i realised the return value to frontend is the same, cos we take `resource.id`, so not doing this